mutator/expression/comparison.go

```diff
@@ -0,0 +1,47 @@
+package expression
+
+import (
+	"go/ast"
+	"go/token"
+	"go/types"
+
+	"github.com/zimmski/go-mutesting/mutator"
+)
+
+var tokenMutations = map[token.Token]token.Token{
```

Comment:
Since this lookup is only used for comparisons, and I do not see it used for something else, please rename it to "comparisonMutations"